source/catalog.html

```diff
@@ -0,0 +1,259 @@
+      <li class="additional-product-list__item">
+        <div class="additional-product-div">
+          <h3 class="additional-product-div__product-name">
+            <a class="additional-product-div__link" href="#">Сахарозаменитель</a>
+          </h3>
+          <dl class="additional-product-div__product-description product-description">
+            <dt class="product-description__product-value">1 упаковка (100 г)</dt>
+            <dd class="product-description__product-quantity">200 Р.</dd>
+          </dl>
+          <a class="additional-product-div__order-button order-button" href="#" role="button">Заказать</a>
+        </div>
+      </li>
+      <li class="additional-product-list__item">
+        <div class="additional-product-div">
+          <h3 class="additional-product-div__product-name"><a class="additional-product-div__link" href="#">Питьевая
+            вода</a>
+          </h3>
+          <dl class="additional-product-div__product-description product-description">
+            <dt class="product-description__product-value">5 литров</dt>
+            <dd class="product-description__product-quantity">50 Р.</dd>
+          </dl>
+          <a class="additional-product-div__order-button order-button" href="#" role="button">Заказать</a>
+        </div>
+      </li>
+      <li class="additional-product-list__item">
+        <div class="additional-product-div">
+          <h3 class="additional-product-div__product-name"><a class="additional-product-div__link" href="#">Молоко</a>
+          </h3>
+          <dl class="additional-product-div__product-description product-description">
+            <dt class="product-description__product-value">1 литр</dt>
+            <dd class="product-description__product-quantity">100 Р.</dd>
+          </dl>
+          <a class="additional-product-div__order-button order-button" href="#" role="button">Заказать</a>
+        </div>
+      </li>
+      <li class="additional-product-list__item">
+        <div class="additional-product-div">
+          <h3 class="additional-product-div__product-name"><a class="additional-product-div__link" href="#">Витамины</a>
+          </h3>
+          <dl class="additional-product-div__product-description product-description">
+            <dt class="product-description__product-value">1 упаковка (30 г)</dt>
+            <dd class="product-description__product-quantity">300 Р.</dd>
+          </dl>
+          <a class="additional-product-div__order-button order-button" href="#" role="button">Заказать</a>
+        </div>
+      </li>
+    </ul>
+    <div class="sale-block">
+      <p class="sale-block__text">Закажите все и получите чехол для кота в подарок!</p>
+    </div>
+  </section>
+  <section class="company-info">
+    <h2 class="visually-hidden">Карта месторасположения офиса "Кэт Энерджи" и адрес для
+      сотрудничества</h2>
+    <div class="company-info__div">
+      <p class="company-info__lead-text">Приглашаем к сотрудничеству дилеров!</p>
```

Comment:
Я еще думала, может ли лид быть модификатором текста, аля: 
```suggestion
      <p class="company-info__text--lead">Приглашаем к сотрудничеству дилеров!</p>
```
но потом проверила их текстовые характеристики - и они слишком разные, так что не имеет смысла в модификаторе, это и правда отдельные элементы